src/EditorFeatures/Test/Workspaces/TestWorkspaceFactory_XmlConsumption.cs

```diff
@@ -225,6 +225,11 @@ public static TestWorkspace CreateWorkspace(string xmlDefinition, bool completed
             if (projectElement.Attribute(FilePathAttributeName) != null)
             {
                 filePath = projectElement.Attribute(FilePathAttributeName).Value;
+                if (string.Compare(filePath, "null", StringComparison.Ordinal) == 0 || string.Compare(filePath, "Nothing", StringComparison.OrdinalIgnoreCase) == 0)
```

Comment:
Yes, I can imagine some unfortunate person in the future investigating some bug with files named "null" with no extension. :stuck_out_tongue: Just make the string a guid. :smile: